### ISSUE TITLE
Bug on Text Widget, overlapped over wrong layer

### ISSUE THREAD

**Issue by szwang-seedtech** [comfyanonymous/ComfyUI#1538]
<img width="629" alt="Screenshot 2023-09-17 195745" src="https://github.com/comfyanonymous/ComfyUI/assets/43228022/e4283e48-e3b9-4fbb-8fe2-576093dac891">

There seens to be a bug on the text widget layer management. When you overlap two nodes both with text widgets, you will find the top node will be blocked by the text widget of the bottom node. Would you mind to fix it?

**Comment by mcmonkey4eva** [comment]
Duplicate of https://github.com/comfyanonymous/ComfyUI/issues/1328
